Index: node_modules/@elastic/plugin-helpers/tasks/build/create_package.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- node_modules/@elastic/plugin-helpers/tasks/build/create_package.js	(revision )
+++ node_modules/@elastic/plugin-helpers/tasks/build/create_package.js	(revision )
@@ -8,7 +8,7 @@
   var buildId = `${plugin.id}-${buildVersion}`;
   var buildSource = plugin.root;
   var buildTarget = join(plugin.root, 'build');
-  var buildRoot = join(buildTarget, 'kibana', plugin.id);
+  var buildRoot = join(buildTarget, plugin.id);
 
   // zip up the package
   return new Promise(function (resolve, reject) {
